src/components/Supporters/index.jsx

```diff
@@ -0,0 +1,227 @@
+import styles from "./styles.module.css";
+import Link from "@docusaurus/Link";
+import ThemedImage from "@theme/ThemedImage";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+/* data of supporters */
+const financial = [
+    {
+        name: "Nvidia",
+        link: "https://www.nvidia.com/",
+        light: "img/supporters/nvidia_light.svg",
+        dark: "img/supporters/nvidia_dark.svg",
+        width: 250,
+    },
+];
+
+const infrastructure = [
+    {
+        name: "Anaconda",
+        link: "https://www.anaconda.com/",
+        light: "img/supporters/anaconda_light.svg",
+        dark: "img/supporters/anaconda_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Open Source Lab",
+        link: "https://osuosl.org/",
+        light: "img/supporters/osl_light.svg",
+        dark: "img/supporters/osl_dark.svg",
+        width: 150,
+    },
+    {
+        name: "OVHcloud",
+        link: "https://www.ovhcloud.com/",
+        light: "img/supporters/ovh_light.svg",
+        dark: "img/supporters/ovh_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Digital Ocean",
+        link: "https://www.digitalocean.com/",
+        light: "img/supporters/do_light.svg",
+        dark: "img/supporters/do_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Oracle Cloud",
+        link: "https://www.oracle.com/cloud/",
+        light: "img/supporters/oracle_light.svg",
+        dark: "img/supporters/oracle_dark.svg",
+        width: 200,
+    },
+];
+
+const developer = [
+    {
+        name: "Anaconda",
+        link: "https://www.anaconda.com/",
+        light: "img/supporters/anaconda_light.svg",
+        dark: "img/supporters/anaconda_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Nvidia",
+        link: "https://www.nvidia.com/",
+        light: "img/supporters/nvidia_light.svg",
+        dark: "img/supporters/nvidia_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Voltron Data",
+        link: "https://voltrondata.com/",
+        light: "img/supporters/voltron_light.svg",
+        dark: "img/supporters/voltron_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Quansight Labs",
+        link: "https://labs.quansight.org/",
+        light: "img/supporters/quansight_light.svg",
+        dark: "img/supporters/quansight_dark.svg",
+        width: 250,
+    },
+    {
+        name: "QuantStack",
+        link: "https://quantstack.net/",
+        light: "img/supporters/quantstack_light.svg",
+        dark: "img/supporters/quantstack_dark.svg",
+        width: 250,
+    },
+];
+
+export default function Supporters() {
+    return (
+        <div className={[styles.supporters, styles.section_padding].join(" ")}>
+            <div className={styles.supporters_conda_forge}>
+                <h1>Supporters</h1>
+                <p>
+                    If you like conda-forge and want to support our mission,
+                    please consider making a{" "}

```

Comment:
What is this strange space thingy? 🤔

Reply:
It is for adding space before and after a word. If we remove that, the highlighted words such as Links and gradient texts would stick to normal texts with no space in between.